pkg/execute/executor.go

```diff
@@ -332,6 +333,61 @@ func (e *DefaultExecutor) runFilterCommand(ctx context.Context, args []string, c
 	return "", errUnsupportedCommand
 }
 
+func (e *DefaultExecutor) runActionCommand(ctx context.Context, args []string, clusterName string) (string, error) {
```

Comment:
As stated in the GH issue, please use new form already:
`@Botkube enable/disable/list action/actions/act <name>` instead of `@Botkube actions enable/disable/list <name>`.

This will be later unified in other commands in the task #703.

BTW I'd recommend splitting the execution part to a separate executor - see the `notifier.go` file for example. In the DefaultExecutor it would be great to have general logic (parsing args, register all the executors, and trigger commands).

Reply:
I tried and the way these values are wired (`arg[1]`) it's very difficult to do this one-by-one. I suggest we do all the commands at the same time.

Reply:
Fair enough 👍 Just please mention it in PR desc 👍 Thanks!

Reply:
@josefkarasek 
On second thought, I think we cannot postpone this, and we need to go with the new command form as we initially planned.
Why? There's a chance we won't complete #703 this release, and as a result, we would do a breaking change for the newly introduced command the next release. If we already know the target form (`enable/disable/list actions/Actions/action/act, then we should support it from day one. We did that the same with `edit SourceBindings` command. 
I looked at the code and I cannot see the difficulty with implementation of it. It shouldn't be a problem, as the `enable/disable/list` verbs are not yet used anywhere else, so there won't be any conflicts. You can see how the `edit sourcebindings` command was implemented:
https://github.com/kubeshop/botkube/blob/main/pkg/execute/executor.go#L207. As you'll dig deeper, we support any kind of casing for the `SourceBindings` name (we make the user input lowercase and we compare it with https://github.com/kubeshop/botkube/blob/main/pkg/execute/edit.go#L34). Let me know how I can help with this.

I won't push too hard on how it is implemented (we can make it more beautiful in #703), but please make sure we support `enable/disable/list actions/Actions <name>` already.  We can introduce aliases like `action` (singular) or `act` later in #703. Thank you!